apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/src/main/java/gov/cms/bfd/pipeline/rda/grpc/apps/RdaServerApp.java

```diff
@@ -58,14 +58,28 @@ public static void main(String[] args) throws Exception {
     LOGGER.info("server stopping.");
   }
 
+  /** The config class to store config options for port, seed, and the number to send to s3. */
```

Comment:
Again, avoid enumeration of fields, which is apt to change and leave this outdated. What this class' responsibility is would just be something like "Configuration details for the RDI server."

Reply:
done